KEGG_parser/parsers.py

```diff
@@ -194,15 +194,16 @@ def split_module_reaction(current_dict, current_entry_name, current_entry_data):
     'COMPOUND': add_module_orthology, 'COMMENT': return_self, 'DBLINKS': add_nested_dict
 }
 
-NOT_CAPTURED_KO_FIELDS = ('REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL', 'SEQUENCE', 'BRITE')
+NOT_CAPTURED_KO_FIELDS = ('REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL', 'SEQUENCE', 'BRITE', 'SYMBOL')
 
 NOT_CAPTURED_RN_FIELDS = ('REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
 
 NOT_CAPTURED_CO_FIELDS = ('BRITE', 'ATOM', 'BOND', 'BRACKET', 'ORIGINAL', 'REPEAT', 'NODE', 'EDGE', 'SEQUENCE',
                           'GENE', 'ORGANISM', 'TYPE', 'EFFICACY', 'PRODUCT', 'CLASS', 'DISEASE', 'TARGET',
-                          'METABOLISM', 'INTERACTION', 'STR_MAP', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
+                          'METABOLISM', 'INTERACTION', 'STR_MAP', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL',
+                          'NETWORK')
 
-NOT_CAPTURED_PATHWAY_FIELDS = ('GENE', 'ORGANISM', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
+NOT_CAPTURED_PATHWAY_FIELDS = ('GENE', 'ORGANISM', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL', 'INCLUDING')
```

Comment:
This makes sense and it looks like it must have been added afterwards. I'm wondering if there is a case where we can capture fields that may get added, but we don't update the not captured fields, just to keep a bit less of a dependency on keeping up with KEGG fields.

Reply:
I agree. Hopefully we can discuss ways to do this during our next code review! @JTFouquier @bsantan @acolorado1 @albatalavera @jboconnor13 (tagging you all so we can discuss at some point)

Reply:
Didn't have a chance to specifically discuss a better fix, but did decide to go ahead and merge.